django-nonrel/ocl/concepts/views.py

```diff
@@ -303,7 +304,11 @@ def get_object(self, queryset=None):
         if self.versioned_object:
             concept_version_identifier = self.kwargs.get(self.lookup_field)
             if not concept_version_identifier:
-                return ConceptVersion.get_latest_version_of(self.versioned_object)
+                concept = self.versioned_object
+                conceptVersion = ConceptVersion.get_latest_version_of(concept)
+                conceptVersion.extras_are_encoded = True
+                conceptVersion.decode_extras()
```

Comment:
Extras are supported not only by Concepts, but also other resources. Ideally we will override the save method in BaseModel to do the encoding and the from_db method to do the decoding, see https://docs.djangoproject.com/en/1.11/ref/models/instances/#customizing-model-loading. Could you please try that approach?

Reply:
I just checked and the from_db method was added in django 1.8 and we are on 1.5 :/ 

I found another solution explained at https://stackoverflow.com/a/33899918 Could you please try that instead?